package.json

```diff
@@ -36,7 +36,7 @@
     "glob-expand": "^0.2.1"
   },
   "peerDependencies": {
-    "typescript": "^2.0.6 || >=2.1.0-dev || >=2.2.0-dev"
+    "typescript": "^2.0.6 || ^2.1.5 || >=2.2.0-dev"
```

Comment:
Is it try with `-g` options?
Unfortunately it is expected behavior.
try with `npm install -g typescript typescript-formatter` in same time. npm don't report that error.